releasenotes/notes/remove-aer-98195057605f1648.yaml

```diff
@@ -0,0 +1,5 @@
+---
+other:
```

Comment:
We do not need a release for this change as this is only unittests or documentation.
Release notes are reserved for end-user facing changes :+1: 

You can remove this file.